.github/workflows/bug_bounty.yml

```diff
@@ -0,0 +1,62 @@
+name: SalesforceCommerceCloud/pwa-kit/bug_bounty
+on:
+  # For testing - to be removed
+  push:
+    branches:
+      - 'mrt-test-ci-deploy'
+  # Run this job when a new Github release is published
+  release:
+    types: [released]
+
+jobs:
+  deploy-bug-bounty:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if latest release
+        id: release
+        run: |-
+          RELEASE_JSON=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/${{ github.repository }}/releases/latest")
+
+          LATEST_TAG_NAME=$(echo "$RELEASE_JSON" | jq -r '.tag_name')
+          RELEASE_EVENT_TAG_NAME=$(echo ${{ github.event.release.tag_name }})
+
+          echo "Latest tag: $LATEST_TAG_NAME"
+          echo "Release event tag: $RELEASE_EVENT_TAG_NAME"
+
+          if [ "$LATEST_TAG_NAME" == "$RELEASE_EVENT_TAG_NAME" ]; then
+            echo "We are releasing the latest. Continuing to deploy to bug bounty instance"
+            echo "IS_LATEST_RELEASE=true" >> $GITHUB_ENV
+          fi
```

Comment:
Testing this is a bit awkward. We either have to produce a new latest release or flip the conditional so that the rest of the flow triggers when LATEST_TAG_NAME != RELEASE_EVENT_TAG_NAME

Here is a run where I reversed the conditional:
https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/13573313405

And here is an early run where I triggered the job by creating a dummy release:
https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/13555776192